changelog.txt

```diff
@@ -1,3 +1,55 @@
+-------------------------------------------------------------------------------
+Version 0.65 - released xxxx 2021
+-------------------------------------------------------------------------------
+# New Features/Enhancements
+* CorsixTH is now available as a Flatpak! (thanks to @rmnvgr !)
+* On MacOS DMG files are now supported as disk images for Theme Hospital data
+* Hospital policies will now carry over to your next level in the campaign
+* You can now remove destroyed rooms for a fee (config option)
+* The information dialogs that appear when a room is first built can be
+  disabled in the configuration file
+* Fire extinguishers in rooms now have a chance to save a smoking machine
+* More adviser messages added
+* New trophy added
```

Comment:
'New' can be interpreted as 'one more of the original game' and as 'a completely new one', you may want to distinguish between those.

Likely previous entry, listing the one new trophy would be nicer.

(and in a sense more useful for history if we want to find back when "foo trophy" was introduced)